bin/sv_varcall_penalties.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+
+from pysam import VariantFile
+import cmdvcf
+import argparse
+from pprint import pprint
+from geneticmodels import GM
+
+MANTA_SIZE_FOR_PENALTY = 100000
+
+def main(args: object):
+    """
+    Read VCF line by line, add genetic models to INFO-field
+    """
+    vcf_object = VariantFile(args.input_vcf)
+    vcf_object.header.info.add(
+        "GQC",
+        ".",
+        "String",
+        "GATK genotype quality score when variant is only called by GATK"
+    )
+    vcf_object.header.info.add(
+        "MANTAPENALTY",
+        ".",
+        "String",
+        "big manta events not supported by gatk/cnvnator"
+    )
+    vcf_out = VariantFile(args.out_vcf, "w", header=vcf_object.header)
```

Comment:
What about putting this into an `output_vcf` function to keep the main function clean